### DISCUSSION TITLE
change the default YAML settings (in VS code)?

### DISCUSSION THREAD

**Discussion by hao-hong** [quarto-dev/quarto-cli#3698] [Q&A]
Hi all, I feel that there is a simple solution but I just cannot find it. 

Each time when I create a new `.qmd` file, my default settings have two lines, one for my title ("untitled") and one for the output (html). How can I change this globally, so that each time when I create a new file, the default settings are my own? I am using VS Code but I am open to other environments.

Thank you.

**Comment by mcanouil** [comment]
Create a code snippet and map it to a keyboard shortcut for example.

There is currently no way to change the default template, this could be a nice feature to add to VSCode extension.
See <https://github.com/quarto-dev/quarto/blob/main/apps/vscode/src/providers/newdoc.ts#L113-L118>.
A new option to provide a path to replace this hardcoded template could be nice.

I am pinging @jjallaire on this.

Note, there is a new repository for the VSCode extension: <https://github.com/quarto-dev/quarto>.

**Comment by cscheid** [comment]
Does `_metadata.yml` on your root directory not work? https://quarto.org/docs/projects/quarto-projects.html#shared-metadata

**Reply by mcanouil** [reply]
I believe he was referring to the "New Quarto document" command which produces a quite limited document:

```
---
title: "Untitled"
format: html
---

```

In RStudio IDE, it's possible to create/override the default templates of any languages.
